sstables/shared_sstable.hh

```diff
@@ -38,22 +38,4 @@ namespace sstables {
 using shared_sstable = seastar::lw_shared_ptr<sstable>;
 using sstable_list = std::unordered_set<shared_sstable>;
 
-std::string to_string(const shared_sstable& sst, bool include_origin = true);
```

Comment:
> No it's unused

agreed. i learned this the hard way after spending almost 2 hours messing around with fmtlib and recompiling the tree ... turns out fmtlib always prefer using `operator<<(ostream&, const seastar::shared_ptr<T>&)` even this one is a better match..

Reply:
Yes, this seastar operator stands on the way :(

Reply:
s/No/Now/ in the changelog